recipes/diffpy.snmf/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{%- set name = "diffpy.snmf" -%}
+{%- set version = "0.1.0" -%}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/d/diffpy_snmf/diffpy_snmf-{{ version }}.tar.gz
+  sha256: 2903bd7d9df588b35bd45bc07ea745c69fb281fb774beef8af645cebcb980b7c
```

Comment:
@xhochy  

We have previously used the jinja syntax for the SHA in all other projects.

https://github.com/conda-forge/diffpy.structure-feedstock/blob/7eba63b44843408c16003f8666b861395d8ce4ed/recipe/meta.yaml#L1-L3

While we made the change as suggested by you, could you please explain why it is a bad practice to use jinja for the sha?

Reply:
We want to minimise the use of Jinja in conda-forge as this has regularly come up as a source of issues with the tooling we have built for conda-forge (mostly in smithy and the autotick bot). Thus if you use a value only once, don't put it in a jinja variable.